katana/wui/core/views.py

```diff
@@ -29,6 +29,11 @@
 from wui.core.core_utils.warrior_recon_creation_class import CreateWarriorRecon
 
 templates_dir = os.path.join(os.path.dirname(__file__), 'templates', 'core')
+try:
+    import django_auth_ldap.backend
+except Exception as e:
+    print(e)
```

Comment:
Seems like this ```try-except``` block has been repeated. Please remove it.

Reply:
Came from Suraj's branch.